tests/backup/backup_upgrade_test.go

```diff
@@ -459,6 +459,21 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", func() {
 			log.InfoD("Upgrading px-backup to latest version [%s]", latestPxBackupVersionFromEnv)
 			err := UpgradePxBackup(latestPxBackupVersionFromEnv)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying completion of px-backup upgrade to latest version [%s]", latestPxBackupVersionFromEnv))
+			// Stork Version will be upgraded on both source and destination if env variable TARGET_STORK_VERSION is defined.
+			latestStorkVersionFromEnv := os.Getenv("TARGET_STORK_VERSION")
```

Comment:
nit: Can you name this variable something like `targetStorkVersion` ?

Reply:
Done